packages/client/src/components/form/SignatureField/SignatureUploader.tsx

```diff
@@ -0,0 +1,260 @@
+/*
```

Comment:
SignatureUploader could be in component library too as it's fairly self contained